packages/ember-application/lib/system/resolver.js

```diff
@@ -269,10 +267,6 @@ export default EmberObject.extend({
     return description;
   },
 
-  makeToString(factory, fullName) {
```

Comment:
This function is not doing anything complicated + second argument is not being used. Can I replace all `makeToString` with `.toString()`?